frontend/src/components/ToggleView.tsx

```diff
@@ -28,18 +28,16 @@ export const ToggleView: React.FC<Props> = ({ enabled, toggle }) => {
 }
 
 const ToggleViewContainer = styled.div`
-  width: 648px;
-  margin: 1.5em;
+  ${tw`m-6`}
 `
 
 const SwitchContainer = styled.span`
-  ${tw`bg-darkGray rounded h-14 m-auto flex border`}
+  ${tw`bg-darkGray relative rounded h-14 m-auto flex border`}
```

Comment:
Oooh nice! Didn't think to make the parent relative! Deals with the hardcoded width :D